src/main/java/com/starkinfra/CreditPreview.java

```diff
@@ -0,0 +1,114 @@
+package com.starkinfra;
+
+import com.starkinfra.utils.Rest;
+import com.starkinfra.utils.SubResource;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+public final class CreditPreview extends SubResource {
+    /**
+     * CreditPreview object
+     * <p>
+     * A CreditPreview is used to get information from a credit before taking it.
+     * This resource can be used to preview credit notes
```

Comment:
Colocar a frase:
* The 'create' function sends the objects to the Stark Infra API and returns the list of preview data.